thorlcr/graph/thgraphmaster.cpp

```diff
@@ -649,6 +649,18 @@ void CMasterActivity::done()
 
 void CMasterActivity::updateFileReadCostStats()
 {
+     // Returns updates numDiskReads & readCost in the file attributes and returns the readCost
```

Comment:
should say  
// Updates numDiskReads & readCost in the file attributes and returns the readCost

?